migration/migrator/migrations/course/20211104173201_queue_message.py

```diff
@@ -0,0 +1,33 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    sql = "ALTER TABLE queue_settings ADD COLUMN IF NOT EXISTS message VARCHAR(400) DEFAULT NULL"
+    database.execute(sql)
+
+def down(config, database, semester, course):
+    """
+    Run down migration (rollback).
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
```

Comment:
Should this column be dropped on rollbacks?

Reply:
I think that'd make sense, I also dropped the message_sent_time on rollback